pkg/reconcilers/deploymentconfig_test.go

```diff
@@ -1078,15 +1078,15 @@ func TestDeploymentConfigProbesMutator(t *testing.T) {
 		wantErr bool
 	}{
 		{
-            name: "Liveness Probe Updated",
-            args: args{
```

Comment:
IDE did this but it looks like the code was in the wrong place anyway. So will leave it